llvm/include/llvm/CodeGen/SelectionDAG.h

```diff
@@ -530,6 +530,9 @@ class SelectionDAG {
 
   /// Pop up a GraphViz/gv window with the DAG rendered using 'dot'.
   LLVM_ABI void viewGraph(const std::string &Title);
+  /// Pop up a GraphViz/gv window with the DAG rendered using 'dot', or write
+  /// the file out instead.
+  LLVM_ABI void viewGraph(const std::string &Title, const bool WriteFile);
```

Comment:
```suggestion
  LLVM_ABI void viewGraph(const std::string &Title, bool WriteFile);
```
const does nothing here